site/src/docs/patterns/cookies/using-the-cookieconsent.mdx

```diff
@@ -0,0 +1,63 @@
+            </ul>
+        </li>
+        <li><strong>Offer the cookie consent banner and cookie settings in all languages that your service supports.</strong></li>
+    </ul>
+</div>
+
+<div class="guideline-dont" style="background: var(--color-error-light); padding: var(--spacing-s); margin-bottom: var(--spacing-layout-2-xs); max-width:600px;">
+    <div class="guideline-dont-label" style="color:var(--color-error); margin-bottom:var(--spacing-s);"><IconError style={{ marginRight: 'var(--spacing-3-xs)', verticalAlign: 'middle'}} size="s"/><strong>Don't</strong></div>
+    <ul style="margin-bottom:0;">
+        <li>The cookie consent banner should not prevent the user from using the web page. Never use an overlay behind the banner or otherwise restrict the use of the site.</li>
+    </ul>
+</div>
+
+## Cookie banner and cookie page
+The HDS CookieConsent features two (2) ways of displaying cookies and cookie settings; banner and page. The banner approach is used to inform the user about cookies and offer a quick way to accept or deny them. The cookie page uses the banner content but renders it as a full page. This page is used to present cookie settings. It also works as a place where the user can later return to review or change cookie settings.
+
+The banner also has a feature to expand to show cookie settings. This expanding settings menu can be used when the amount of cookies is low, e.g. less than five (5) individual cookies. Otherwise, it is recommended to open the full settings page. It is easier for the user to browse and manage.
+
+No matter whether you use expanding cookie banner or not, you must always offer a full cookie settings page to the user. It is required to allow the user to review and edit cookie settings after the initial consent has been given. Place a link to the cookies page on the page footer element.
+
+## How does the cookie consent banner affect other content of the service?
+There may be elements or content on your service that do not work properly until the user has given consent to optional cookies. It is recommended to either hide this content or show clear error messages at the position of these elements that indicate why they are not available.
+
+## What happens after the user has accepted or denied the optional cookies?
+The user must be able to view and edit the cookie consent after they have initially submitted their choices. You must offer a separate cookies page that allows changing viewing the cookies and changing consent. The link to this page should be placed on the footer element of the page. Note that the HDS CookieConsent component offers functionality to render it as a page. This allows you to use the same component and data to construct a separate cookie page.
+
+If new cookies are added or current ones changed, consent should be asked from the user again the next time the service is used.
```

Comment:
We should probably provide an example of how to technically change cookie in a way that it would trigger the cookie consent modal and show the cookie as not accepted. Because currently it is id-based, so if the id stays the same, the component won't notice that the cookie has changed.

Reply:
Maybe it is enough just to say: "current ones changed" -> "ids/names of the current ones are changed"

Reply:
Sorry, the "id" of the cookie must be changed to show the modal again. Changing the "name" has no effect as it is just text in the UI.